specification/storagesync/resource-manager/Microsoft.StorageSync/2017-06-05-preview/storagesync.json

```diff
@@ -0,0 +1,3232 @@
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "CloudEndpoint Resource"
+        ],
+        "operationId": "CloudEndpointsGet_ListBySyncGroup",
+        "description": "Get a CloudEndpoint List.",
+        "x-ms-examples": {
+          "CloudEndpointsGet_ListBySyncGroup": {
+            "$ref": "./examples/CloudEndpointsGet_ListBySyncGroup.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Array of Cloud Endpoint resources in Sync Group",
+            "schema": {
+              "$ref": "#/definitions/CloudEndpointArray"
+            },
+            "headers": {
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error message indicating why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/storageSyncServices/{storageSyncServiceName}/syncGroups/{syncGroupName}/cloudEndpoints/{cloudEndpointName}/prebackup": {
+      "post": {
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "storageSyncServiceName",
+            "in": "path",
+            "description": "Name of Storage Sync Service resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "syncGroupName",
+            "in": "path",
+            "description": "Name of Sync Group resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "cloudEndpointName",
+            "in": "path",
+            "description": "Name of Cloud Endpoint object.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Body of Backup request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BackupRequest"
+            }
+          }
+        ],
+        "tags": [
+          "CloudEndpoint Resource",
+          "Actions",
+          "Backup Restore"
+        ],
+        "operationId": "CloudEndpointPreBackup_Post",

```

Comment:
Operation ID should be in the format `"group_verb"`. So this should be changed to something like:
`"operationId": "CloudEndpoints_PreBackup"`

Reply:
resolved